docs/user-guide/index.md

```diff
@@ -57,6 +57,9 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## Permission Issue
```

Comment:
@alrra Done. One question - some links can get quite long in markdown. Do we always have to use inline links? Referenced links seem to work better in those situations.